lib/gui/pages/main/controllers/flash.js

```diff
@@ -57,12 +60,15 @@ module.exports = function(
     // otherwise Windows throws EPERM
     DriveScannerService.stop();
 
+    const imagePath = image.path;
+
     AnalyticsService.logEvent('Flash', {
-      image: image,
-      device: drive.device
+      image,
+      drive,
+      unmount: SettingsModel.get('unmountOnSuccess')
     });
 
-    ImageWriterService.flash(image, drive).then(() => {
+    ImageWriterService.flash(imagePath, drive).then(() => {
```

Comment:
Can we do `image.path` here, since its the only place where the `imagePath` variable is used?